src/cursor/cur_backup.c

```diff
@@ -650,18 +648,15 @@ __backup_config(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, const char *cfg[
 
         /*
          * Handle log targets. We do not need to go through the schema worker, just call the
-         * function to append them. Set log_only only if it is our only URI target.
+         * function to append them.
          */
         if (WT_PREFIX_MATCH(uri, "log:")) {
             log_config = true;
-            *log_only = !target_list;
             WT_ERR(__backup_log_append(session, session->bkp_cursor, false));
         } else if (is_dup)
             WT_ERR_MSG(
               session, EINVAL, "duplicate backup cursor cannot be used for non-log target");
```

Comment:
"non-log" is strange wording when we no longer support log-based incremental backup. Is there a better way to describe this?

Reply:
It is correct and not something new or changed.